Podfile

```diff
@@ -158,6 +158,7 @@ target 'WordPress' do
         shared_with_all_pods
         shared_with_networking_pods
         wordpress_ui
+        pod 'ZIPFoundation', '~> 0.9.8'
```

Comment:
@nheagy, does `ZIPFoundation` need to be listed in the  `WordPressDraftActionExtension` extension below 👇 ?

Reply:
I _don't think so_ but I'll try to double check.

Reply:
It seems to build fine 😉